rising/transforms/spatial.py

```diff
@@ -15,9 +16,9 @@
 schduler_type = Callable[[int], Union[int, Sequence[int]]]
 
 
-class Mirror(RandomDimsTransform):
-    def __init__(self, dims: Sequence, keys: Sequence = ('data',),
-                 prob: Union[float, Sequence] = 0.5, grad: bool = False, **kwargs):
+class Mirror(BaseTransform):
+    def __init__(self, dims: Union[int, AbstractParameter, Sequence],
```

Comment:
we should add an integration for each case here. (mirror is a pretty easy example to test the injection with float and Sequence)